File: android/hexagonal-architecture-with-compose/app/src/main/java/ar/com/composehexagonalarchitecture/domain/Team.kt
package ar.com.composehexagonalarchitecture.domain

data class Team (
    val abbreviation: String,
    val city: String,
    val conference: String,
    val division: String,
    val fullName: String,
    val id: Int,
    val name: String,
    val logo: String? = null,
)

File: ios/base/Demo/Demo/Data/Domain/Team.swift
//
//  Team.swift
//  Demo
//
//  Created by Marco Scabbiolo on 09/06/2022.
//

import Foundation

struct Team: Decodable {
    let abbreviation: String
    let city: String
    let conference: String
    let division: String
    let fullName: String
    let id: Int
    let name: String
    let logo: String?
}
